android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/network/Uris.java

```diff
@@ -92,16 +90,10 @@ public Uris(Context context) {
                             // Parse out each line of the index file and split them into batches as indicated by
                             // the leading timestamp in the filename, e.g. "1589490000" for
                             // "exposureKeyExport-US/1589490000-00002.zip"
-                            final String zipFileName =  RealmSecureStorageBte.INSTANCE.getLastDownloadedKeyZipFileName();
-                            final int startIndex = getStartIndex(indexEntries, zipFileName);
-                            for (int i = startIndex; i < indexEntries.size(); i++) {
+                            // In case that we want to limit the number of downloaded files we can add some logic to check the latest file name here
+                            for (int i = 0; i < indexEntries.size(); i++) {
                                 final String indexEntry = indexEntries.get(i);
                                 uriList.add(baseDownloadUri.buildUpon().appendEncodedPath(indexEntry).build());
-
-                                if(uriList.size() == MAX_KEY_BATCHES_PER_DAY) {
```

Comment:
This is not needed anymore
According to the documentation there is only a limit of 6 calls to `provideDiagnosisKeys()` per day, but there is no limit on the number files we provide.
It won't have any impact since we only call the method once a day.
https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods